cf_remote/ssh.py

```diff
@@ -57,13 +57,17 @@ def connect_wrapper(host, *args, **kwargs):
     return connect_wrapper
 
 
-def scp(file, remote, connection=None):
+def scp(file, remote, connection=None, rename=None):
     if not connection:
         with connect(remote) as connection:
-            scp(file, remote, connection)
+            scp(file, remote, connection, rename)
     else:
         print("Copying: '{}' to '{}'".format(file, remote))
```

Comment:
```Python
print(f"Copying: '{file}' to '{remote}'")
```
Would make it a tiny bit more neat, from my point of view

Reply:
Yep, but IIRC this part was written before we had f-strings :)

Reply:
https://github.com/cfengine/cf-remote/pull/8